### ISSUE TITLE
[Suggestion] Mod Compatability for Armor

### ISSUE THREAD

**Issue by vanwertusername** [AlexModGuy/Ice-and-Fire-Suggestions#431]
Is there a plan to add mod compatibility with dragon armor? Mostly looking at Thaumcraft because I've been into it the most. I'd love to see some Thaumium dragon armor.

Edit: Made it a bit clearer to understand what I was asking

**Comment by Xorbah** [comment]
What do you mean by "compatibility" here?

**Comment by vanwertusername** [comment]
As in, having dragon armors made of diffetent materials...

**Comment by Xorbah** [comment]
I believe that's already in the works by an addon.

**Comment by vanwertusername** [comment]
Is that just a guess? I did some searching but didn' find anything

**Comment by Xorbah** [comment]
No, it's not a guess. There's a project called dark bestiary that's doing
it and last I checked I don't think new armors (with the exception of
silver) has been planned for the main mod.

**Comment by vanwertusername** [comment]
Ahhh, thanks

**Comment by vanwertusername** [comment]
Could you provide a link please?

**Comment by Xorbah** [comment]
Do you use discord?

**Comment by vanwertusername** [comment]
Yes

**Comment by Xorbah** [comment]
https://discord.gg/nndQT5
